spec/mailers/first_draft_collections_mailer_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe FirstDraftCollectionsMailer, type: :mailer do
-  let(:creator) { build_stubbed(:user, name: 'Peter Lorre', first_name: 'Snaaaaakes') }
+  let(:creator) { build_stubbed(:user, name: 'Peter Lorre', first_name: 'Snaaaaakes', email: 'psl@example.org') }
```

Comment:
can we get rid of `first_name` now? I don't think this is useful anymore.

Reply:
yes - good catch.